CONTRIBUTING.md

```diff
@@ -31,6 +31,31 @@ use GitHub pull requests for this purpose. Consult [GitHub
 Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+### Contribution guidelines and standards
+
+Before sending your pull request for 
+[review](https://github.com/tensorflow/swift-apis/pulls), 
+make sure your changes are consistent with the guidelines and follow the
+TensorFlow coding style.
+
+#### General guidelines and philosophy for contribution
+
+*   Include unit tests when you contribute new features, as they help to a)
+    prove that your code works correctly, and b) guard against future breaking
+    changes to lower the maintenance cost.
+*   Bug fixes also generally require unit tests, because the presence of bugs
+    usually indicates insufficient test coverage.
+
+#### License
+
+Include a license at the top of new files.
+* [License example](https://github.com/tensorflow/swift-apis/blob/master/Sources/TensorFlow/Random.swift)
+
+#### Swift Coding Style
+
+Changes to Swift for Tensorflow code should conform to
+[Google Swift Style Guide](https://google.github.io/swift/).
```

Comment:
Both [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/) and [Google Swift Style Guide](https://google.github.io/swift/) with the exception that 4-space indentation should be used.